### PR TITLE
avocado-juice recipe

### PR DESCRIPTION
**Pull request by misspuff55**

### CODE REVIEW

File: src/recipes/avocado-juice.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Avocado Juice",
+    "description": "The avocado, also known as butter pear or alligator pear, is a fruit that is widely acknowledged to have properties that reduce cholesterol levels in the body. To top it off, it's absolutely delicious. so this is avocado juice recipe",
+    "github": "misspuff55",
+    "ingredients": [
+        "5 Tablespoons sugar",
+        "5 Tablespoons water",
+        "2 avocados, peeled and pit removed",
+        "½ cup milk,
```
Comment (alfg) [inline src/recipes/avocado-juice.json]:
```suggestion
        "½ cup milk",
```

Also, can you move the images to the new location at `src/assets/recipes/`?

Comment (misspuff55) [reply src/recipes/avocado-juice.json]: done sir, thanks btw

**Comment by alfg** [comment]
Please see the updated ingredients format and images location:
https://github.com/alfg/opendrinks/blob/master/CONTRIBUTING.md

**Comment by alfg** [comment]
Closing. Feel free to reopen with the updated JSON format. Thanks.

**Pull request closed without merging**